fhir-parser-resources/template-resource.py

```diff
@@ -1,5 +1,5 @@
-#  Generated from FHIR {{ info.version }} ({{ profile.url }}) on {{ info.date }}.
-#  {{ info.year }}, SMART Health IT.
+# Generated from FHIR {{ info.version }} ({{ profile.url }}).
+# {{ info.year }}, SMART Health IT.
```

Comment:
I figured including the timestamp of generation made reviews so noisy, it wasn't worth it. Year I'll keep for copyright reasons. (Though I'm not 100% on the law around copyright of these generated files necessarily being the year of generation... but whatever, not my lane.)

Reply:
could i talk you into subbing in the version of the fhir client? is that useful?

Reply:
Hmm OK but... the version of fhirclient at generation time might not be the version at release time (i.e. we'd have to add a release-time check item to regenerate models with the correct version).

As for whether it is useful... it's got a non-zero utility. The version could be seen in `__init__.py` a folder up, but that doesn't mean it's bad to repeat it here (except for the risk of being stale, as above).

What's your vibe on the tradeoff? We could add a unit test to check the comment against the version so they don't go stale...